docs/04_protocol/index.md

```diff
@@ -16,9 +16,9 @@ The core protocols are:
 
 ## System
 
-The Antelope blockchain framework is unique in that the features and characteristics of the blockchain built on it are flexible, that is, they can be changed, or be modified completely to suit each business case requirement. Core blockchain features such as consensus, fee schedules, account creation and modification, token economics, block producer registration, voting, multi-sig, etc., are implemented inside smart contracts which are deployed on the blockchain built on the Antelope framework. These smart contracts are referred to as `system contracts` and the layer as the `Antelope system` layer, or simply `system` layer.
+The EOS blockchain is unique in that the features and characteristics of the blockchain built on it are flexible, that is, they can be changed, or be modified completely to suit each business case requirement. Core blockchain features such as consensus, fee schedules, account creation and modification, token economics, block producer registration, voting, multi-sig, etc., are implemented inside smart contracts which are deployed on the blockchain built on the EOS blockchain. These smart contracts are referred to as `system contracts` and the layer as the `EOS system` layer, or simply `system` layer.
```

Comment:
which are deployed on the blockchain built on the EOS blockchain --> which are built and deployed on the EOS blockchain